test/unit/src/math/Color.tests.js

```diff
@@ -4,14 +4,21 @@ import { Color } from '../../../../src/math/Color.js';
 import { ColorManagement } from '../../../../src/math/ColorManagement.js';
 import { eps } from '../../utils/math-constants.js';
 import { CONSOLE_LEVEL } from '../../utils/console-wrapper.js';
-import { DisplayP3ColorSpace, SRGBColorSpace } from '../../../../src/constants.js';
+import { SRGBColorSpace } from '../../../../src/constants.js';
+import { DisplayP3ColorSpace, DisplayP3ColorSpaceImpl } from '../../../../examples/jsm/math/ColorSpaces.js';
```

Comment:
Can we remove any addons-related unit tests from `test/unit/src`?

You might want to add tests in the `test/unit/addons` directory instead.

Reply:
Done! ✅